Fixed UI tags for Network Steering config header

brian-silabs opened the pull request:
Fixed an issue with the UI tags of the Network Steering Component configuration header for options :
* RADIO_TX_POWER
* CHANNEL_MASK

This was displaying those options as toggle switches instead of input boxes in Simplicity Studio config UI

MrMossevig commented:
Thank you for contributing!

As you know, this repository is a push-only repository and we do not accept pull-requests, as we need to keep the releases synchronized across all our distribution channels. However, we still like that you use pull requests for notifying us about potential code improvements so continue to do that in the future. We have recorded the code changes and will seek to include this in the upcoming SDK-release.

nestorayuso commented:
Hi again, 
This repository (v5.0.0) is outdated, i can see in simplicity studio the new v5.1.2, and it is located in different place under gecko_sdk_suite folder, I don't know why

MrMossevig commented:
Hi,

You're correct. v5.1.2 was released in Simplicity Studio on Friday March 10th. We're just in the process of updating GitHUb with the latest version.

New versions have always had their own folder, but we have moved it from the MCU-specific **exx32** to **gecko_sdk_suite** because it's now not only available for MCUs, but you can get the entire range of wireless stacks (properitary, zigbee, thread, bluetooth) installed in the same directory to simplify development across different wireless standards.

TabithaParker commented:
This repository is now updated to v5.1.2.

brian-silabs committed:
Fixed UI tags for Network Steering config header

Fixed an issue with the UI tags of the Network Steering Component configuration header for options :
* RADIO_TX_POWER
* CHANNEL_MASK

This was displaying those options as toggle switches instead of input boxes in Simplicity Studio config UI

brian-silabs committed:
Merge branch 'SiliconLabs:gsdk_4.0' into gsdk_4.0
